roles/logins/tasks/main.yml

```diff
@@ -157,4 +157,11 @@
   when:
     - ansible_facts['os_family'] == "RedHat"
     - ansible_facts['distribution_major_version'] >= "8"
+
+- name: Extend pam.d/sshd to limit limit user ssh resources
```

Comment:
```suggestion
- name: Extend PAM stack to limit resources for users that login via SSH.
```

Reply:
keeping my option because it clearly defines which file is changed
pam.d/sshd is a shortened version of /etc/pam.d/sshd - but the entire path is unneeded